### ISSUE TITLE
Load/Save files using DurexForth CRT 

### ISSUE THREAD

**Issue by Vagant1** [jkotlinski/durexforth#521]
Hi,
I would like to use the CRT file with my Ultimate 2+ to speed up loading of the Forth on c64 but I don't know how I could configure the source/destination to read files? The LS command comes with read/write error. Is there any way around this?

**Comment by jkotlinski** [comment]
Hi!
Is this with master or version 4.0.0?
You may need to type ”8 device” to choose disk drive.

tis 7 feb. 2023 kl. 09:49 skrev Vagant1 ***@***.***>:

**Comment by Vagant1** [comment]
Thanks for quick response. I am using Durexforth-v4_0_0.crt from last release. When I try 8 device and then ls I am receiving error: 74, drive not ready,00,00

**Comment by jkotlinski** [comment]
That means there is no disk inserted in the drive.

tis 7 feb. 2023 kl. 10:44 skrev Vagant1 ***@***.***>:

**Comment by Vagant1** [comment]
Correct, I am using Ultimate 2+ with my C64 and reading files of the USB stick. Same with Vice. If I use d64 it is working well with no disc but what you saying is with cartridge file i.e. CRT it really needs a disc drive attached?

**Comment by jkotlinski** [comment]
Have you mounted a .d64 image?
If not, it will give the drive not ready error, when trying to list the
files.

tis 7 feb. 2023 kl. 12:01 skrev Vagant1 ***@***.***>:

**Comment by Vagant1** [comment]
Yes, I have no problem with listing and writing files using d64 image - all is working great. The problem is when I try to use the CRT image. Not a major issue but would be helpful - much faster than loading from the image.

**Comment by jkotlinski** [comment]
I am not sure that I follow.
What behavior would you expect from the ”ls” command when no disk image is
attached?
What should it show?

tis 7 feb. 2023 kl. 12:07 skrev Vagant1 ***@***.***>:

**Comment by Vagant1** [comment]
All working fine.

**Comment by jkotlinski** [comment]
OK, happy to hear it worked out for you!
